tasks/download.sh

```diff
@@ -36,7 +36,18 @@ verify-file() {
 
 download() {
   printf '%s\n' "Downloading: ${1}"
-  curl -s -f -L -o "$2" "$1"
+  tmp_file=$(mktemp "peadm-download.XXX")
```

Comment:
Did some testing; if you give `mktemp` any args, it breaks the default file location/pathing and ONLY uses the template; e.g., it won't use the system default tempdir correctly. So, we shouldn't give it any args.
```suggestion
  tmp_file=$(mktemp)
```